README.md

```diff
@@ -12,60 +12,60 @@ platform.
 
-- Q: A general push notification platform for all systems? How does this differ
-  from services like [urbanairship](http://urbanairship.com)?
-- A: [urbanairship](http://urbanairship.com) is a great service, and there're
-  other similar services available, like [openpush](http://openpush.im/),
-[notifica](https://notifica.re/), etc. All of them are wonderful services.
-However, [uniqush](http://uniqush.org) is different from them.
+- Q: Is this a general push notification platform for all types of devices? How does this differ
+  from services such as [Urban Airship](http://urbanairship.com)?
+- A: [Urban Airship](http://urbanairship.com) is a great service, and there are
+  other similar services available, like [OpenPush](http://openpush.im/),
+[Notificare](https://notifica.re/), etc. All of them are wonderful services.
+However, [Uniqush](http://uniqush.org) is different from them.
 [Uniqush](http://uniqush.org) is not a service. Instead,
 **[Uniqush](http://uniqush.org) is a system, which runs on your own
-server**. In fact, if you wish, you can use uniqush to setup a similar service like [urbanairship](http://urbanairship.com).
+server**. In fact, if you wish, you can use Uniqush to setup a service similar to [Urban Airship](http://urbanairship.com).
 
 - Q: OK. Then is it a library? Like
   [java-apns](https://github.com/notnoop/java-apns)?
-- A: Well.. Not actually. I mean, it is a program, like apache httpd. You
-  download it, you run it, that's it. You don't need to worry about which
-language to use, package dependencies, etc. 
+- A: Well.. Not actually. I mean, it is a program, like Apache HTTP Server. You download it, you run it. It does require a [Redis](http://redis.io/) server, but, other than that, you don't need to worry about which language to use, package dependencies, etc. 
 
-- Q: But wait, how can I use it anyway? I mean, if my program wants to push
-  something, I need to tell uniqush about this action. How can I communicate
-with uniqush? There must be some library so that I can use it in my
-program to talk with uniqush, right?
-- A: We are trying to make it easier. *uniqush-push* provides RESTful APIs. In
-  other words, you talk with *uniqush-push* through HTTP protocol. As long as
+- Q: But wait, how can I use it anyway? I mean, if my program wants to send
+  a push notification, I need to tell Uniqush about this action. How can I
+  communicate with Uniqush? There must be some library so that I can use it
+  in my program to talk with Uniqush, right?
+- A: We are trying to make it easier. `uniqush-push` provides RESTful APIs. In
+  other words, you talk with `uniqush-push` through HTTP protocol. As long as
 there's an HTTP client library for your language, you can use it and talk with
-*uniqush-push*. For details about the our RESTful APIs, you may want to check
+`uniqush-push`. For details about the our RESTful APIs, you may want to check
 out our [document](http://uniqush.org/documentation/index.html).
```

Comment:
"out our document" -> "out our documentation"

Also, instead of documentation, it might be better to say `[our API documentation](http://uniqush.org/documentation/basic-opts.html)`

"you may want to check out" -> "check out"/"see"